### ISSUE TITLE
A11Y: Form Accessibility

### ISSUE THREAD

**Issue by tisto** [plone/volto#1505]
Make Volto forms accessible. Placeholder issue.

**Comment by ThomasKindermann** [comment]
PR for updated Test: 
https://github.com/plone/volto/pull/2569

Errors from A11y test: 
- Contact Form doesn't have a component with the "#view" id so the a11y test fails
- color contrast for active form text is too low, either change to black or make it a darker grey

**Comment by nsundriyal62** [comment]
can i work on this issue?
should i directly generate a pr or you gonna assign this to me?

reagards !

**Comment by stevepiercy** [comment]
@nsundriyal62 please read and follow [Contributing to Plone](https://6.docs.plone.org/contributing/index.html) and [First-time contributors](https://6.docs.plone.org/contributing/first-time.html). There is already an open pull request. Additionally the description lacks sufficient information to take action. @tisto what exactly needs to be improved, and have other recent issues with details superseded this one? Maybe we close this one?

**Comment by nsundriyal62** [comment]
Ok sir

I will go through documentation First. 
Thanks for replying.

**Comment by davisagli** [comment]
Fixed in https://github.com/plone/volto/pull/6307
